Atomic/info.py

```diff
@@ -66,20 +66,29 @@ def __init__(self):
         super(Info, self).__init__()
         self.beu = BackendUtils()
 
+    def version_tty(self):
+        if self.args.debug:
+            util.write_out(str(self.args))
+        util.write_out(self.version())
```

Comment:
Oh yeah, using stringio is better than `+=`.  My comment above builds on that - we'd just use `sys.stdout` for ttys.